cam0:
  cam_overlaps: [1]
  camera_model: pinhole
  distortion_coeffs: [0.3794326297071376, 0.5478913245079665, -2.852649894644302,
    3.9247347043349556]
  distortion_model: equidistant
  intrinsics: [901.868506540884, 901.7226616928423, 658.1814805073678, 352.0009920796922]
  resolution: [1280, 720]
  rostopic: /camera/color/image_raw
cam1:
  T_cn_cnm1:
  - [0.9999999999995391, 5.971797626797458e-09, -9.661733598217424e-07, -1.4379185998242093e-06]
  - [-5.9724373981475945e-09, 0.9999999999997867, -6.621703467125873e-07, -1.3905571029476686e-07]
  - [9.661733558671832e-07, 6.621703524826882e-07, 0.99999999999932, 5.1882470729204066e-06]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [0.3792204180566189, 0.5500650905738175, -2.8608101140998294,
    3.934778116655062]
  distortion_model: equidistant
  intrinsics: [901.8766340039763, 901.7312010590206, 658.1832851362336, 352.00171363332845]
  resolution: [1280, 720]
  rostopic: /camera/color/image_raw
